src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -904,13 +927,88 @@ private JButton getCleanTableButton() {
 				selectAll = false;
 				specificRadio.setSelected(true);
 				codeTextField.setEnabled(true);
-				inventoryRowSearchList.clear();
-				fireInventoryUpdated();
+				inventoryRowSearchList = new ArrayList<>();
+				DefaultTableModel model = (DefaultTableModel) jTableInventoryRow.getModel();
+				model.setRowCount(0);
+				model.setColumnCount(0);
+				jTableInventoryRow.updateUI();
+				ajustWith();
```

Comment:
also in InventoryBrowser.java

```suggestion
				adjustWith();
```